lookout/style/format/features.py

```diff
@@ -57,9 +57,12 @@ def __str__(self):
         return self.value
 
     def __repr__(self):
-        return "VirtualNode(\"%s\", start=%s, end=%s, node=%s, path=\"%s\")" % (
-            self.value, tuple(self.start), tuple(self.end),
-            id(self.node) if self.node is not None else "None", self.path)
+        return "VirtualNode(\"%s\",%s start=%s, end=%s, node=%s, path=\"%s\")" % (
+            self.value.replace("\n", "\\n"),
+            "" if self.y is None else " y=%s," % self.y,
```

Comment:
When node is None, we output "None" so I think we should also do that for y for consistency.